Resources/services.xml

```diff
@@ -19,7 +19,7 @@
 
         <service id="frosh_tinypng_optimizer.config" class="Shopware\Components\Plugin\CachedConfigReader">
             <factory service="shopware.plugin.cached_config_reader" method="getByPluginName"/>
-            <argument type="string">FroshTinyPngMediaOptimizer</argument>
+            <argument type="string">%frosh_tiny_png_media_optimizer.plugin_name%</argument>
```

Comment:
This was implemented with SW 5.2.10.
Plugin should be compatible by 5.2.0

could you please revert or set the parameter on Plugin.build?

Reply:
Hehe. You defined 5.2.17 as a minimum ;)